modules/custom/az_google_tag/az_google_tag.module

```diff
@@ -15,7 +15,7 @@ function az_google_tag_toolbar_alter(&$items) {
     $site_url = \Drupal::request()->getHost();
     $items['az_quickstart']['tray']['az_links']['#links']['analytics'] = [
       'title' => t('Analytics Dashboard'),
-      'url' => Url::fromUri('https://datastudio.google.com/u/0/reporting/938c3f2a-6dbf-4968-9edd-8963450b4200/page/kTLWB', [
+      'url' => Url::fromUri('https://lookerstudio.google.com/reporting/ef02c272-afe4-4862-b4bd-8706e3436b2f', [
         'query' => [
           'params' => '{"df62":"include%EE%80%800%EE%80%80IN%EE%80%80' . $site_url . '/"}',
```

Comment:
The existing version of this link in Quickstart (that links to the old analytics dashboard) automatically pre-selects the correct subdomain based on the domain of the site that rendered the link.

I tested replacing the ProboCI site domain in the generated link with `quickstart.arizona.edu` but found that it did not pre-select the correct subdomain as expected.

I think if we make these changes to the URL the subdomain auto selection functionality would work as expected (on sites with real subdomains -- not ProboCI).

```suggestion
      'url' => Url::fromUri('https://lookerstudio.google.com/reporting/ef02c272-afe4-4862-b4bd-8706e3436b2f/page/nbaTD', [
        'query' => [
          'params' => '{"df21":"include%EE%80%800%EE%80%80IN%EE%80%80' . $site_url . '/"}',
```
Example generated link following this pattern:
https://lookerstudio.google.com/u/0/reporting/ef02c272-afe4-4862-b4bd-8706e3436b2f/page/nbaTD?params=%7B%22df21%22:%22include%25EE%2580%25800%25EE%2580%2580IN%25EE%2580%2580quickstart.arizona.edu%22%7D